cogs/gamerwords.py

```diff
@@ -267,8 +294,21 @@ async def handle_new_gamer_message(self, message):
 				allowed_mentions=allowed_mentions
 			)
 
+	async def dl_attach(self, attach, message):
+		if attach.size >= getattr(message.guild, 'filesize_limit', 8 * 1024 ** 2):
+			await asyncio.sleep(0.2)
+			await message.delete()
+			raise Break
+
+		file = await attach.to_file()
+		if self.has_gamer_words(file.filename):
+			file.filename = self.censor_gamers(file.filename)
+
+		return file
+
 	def censor_gamers(self, string):
-		return self.replacer.replace(string)
+		replacer = UnicodeAwareBruhReplacer(string)
```

Comment:
like i said, just inline this and delete censor_gamers. it doesn't need to be two lines.
```suggestion
    UnicodeAwareBruhReplacer(string).replace()
```